### ISSUE TITLE
Uncaught TypeError: this.element.querySelector is not a function

### ISSUE THREAD

**Issue by ghost** [coronalabs/CoronaSDK-AtomPlugin#4]
[Enter steps to reproduce:]

1. ...
2. ...

**Atom**: 1.19.0-beta2 x64
**Electron**: 1.6.9
**OS**: TEAMOS Windows 7 elementary 
**Thrown From**: [autocomplete-corona](https://github.com/coronalabs/CoronaSDK-AtomPlugin) package 1.0.7

### Stack Trace

Uncaught TypeError: this.element.querySelector is not a function

```
At C:\Users\DANESH\.atom\packages\autocomplete-corona\node_modules\atom-package-deps\lib\view.js:41

TypeError: this.element.querySelector is not a function
    at View.advance (/packages/autocomplete-corona/node_modules/atom-package-deps/lib/view.js:41:36)
    at /packages/autocomplete-corona/node_modules/atom-package-deps/lib/main.js:53:16
    at stdout (/packages/autocomplete-corona/node_modules/atom-package-deps/lib/helpers.js:24:11)
    at Socket.<anonymous> (~/AppData/Local/atom/app-1.19.0-beta2/resources/app/src/buffered-process.js:212:17)
    at emitOne (events.js:96:13)
    at Socket.emit (events.js:188:7)
    at readableAddChunk (_stream_readable.js:176:18)
    at Socket.Readable.push (_stream_readable.js:134:10)
    at Pipe.onread (net.js:551:20)
```

### Commands

```
     -2:20.9.0 core:backspace (input.hidden-input)
     -2:13.7.0 core:confirm (input.hidden-input)
     -1:58.3.0 core:backspace (input.hidden-input)
     -1:58.3.0 core:delete (input.hidden-input)
  2x -1:58 core:confirm (input.hidden-input)
```

### Non-Core Packages

```
autocomplete-corona 1.0.7 
language-lua 0.9.11 
```

**Comment by myisaak** [comment]
I have the same issue too!

**Comment by zerodarkzone** [comment]
I have the same issue

**Comment by myisaak** [comment]
Once I restarted it, the plugin worked.

**Comment by Shchvova** [comment]
This error message only to show that dependency `language-lua` was installed correctly and can be safely disregarded.

**Comment by Shchvova** [comment]
This issues is caused by used dependency, steelbrain/package-deps#108

**Comment by Shchvova** [comment]
This is fixed in 1.1.3
